app/js/components/LabOrdersList.jsx

```diff
@@ -444,7 +444,7 @@ export class LabOrdersList extends PureComponent {
       headerClassName: `lab-order-list-column-header lab-order-list-header-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
     }));
     return (
-      <div className="lab-order-list">
+      <div className="lab-order-list" data-cy="order-list">
```

Comment:
interesting, is this a cypress-test specific tag?

Reply:
Yeah, see [Best Practices](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements).